packages/fetchai/skills/weather_client/dialogues.py

```diff
@@ -36,18 +38,26 @@
 class Dialogue(FipaDialogue):
     """The dialogue class maintains state of a dialogue and manages it."""
 
-    def __init__(self, dialogue_label: DialogueLabel, is_seller: bool) -> None:
+    def __init__(self, dialogue_label: DialogueLabel) -> None:
         """
         Initialize a dialogue label.
 
         :param dialogue_label: the identifier of the dialogue
-        :param is_seller: indicates whether the agent associated with the dialogue is a seller or buyer
 
         :return: None
         """
-        FipaDialogue.__init__(self, dialogue_label=dialogue_label, is_seller=is_seller)
+        FipaDialogue.__init__(self, dialogue_label, FipaDialogue.AgentRole.BUYER)
```

Comment:
Why hard code it? I'd prefer we implement `role_from_first_message` properly below

Reply:
Done